docs/hadoop/hdfs_service.md

```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
 ### Data Node <a name="Data_Node"></a>
 
 * Configuration Data: Its path is defined by *hadoop-data-node-configuration* configuration map.
-* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, otherwise in the *hdfs/data* directory under the storage path.
+* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, 
+                    otherwise in the *hdfs/data* directory under common storage path.
 * Host Configuration: Its path is defined by *host-configuration* configuration map.
 * Temp Data: It is in the *hadooptmp/datanode* directory under the storage path.
 
+## Data Migration <a name="Data_Migration"></a>
+
+If you want to reconfigure HDFS data storage path, please follow these steps to avoid data loss:
+1. Confirm HDFS path, it will be *${hadoop-data-node.storage_path}* if specified, 
+    or *${cluster.common.data-path}/hdfs/data*. Assume current cluster HDFS path is *path1*, 
+    new path is *path2*.
+2. Stop hadoop data node.
+    ```bash
+    python paictl.py service stop -n hadoop-data-node
+    ```
+3. On every data node, manually copy data from *path1* to *path2*.
+4. Edit cluster-configuration *${hadoop-data-node.storage_path}* to *path2*, 
```

Comment:
pls tell user in which config file and link direct to that file.  https://github.com/Microsoft/pai/blob/6e6d41da7a52fdcf5666d559281441db78c6472c/deployment/quick-start/services-configuration.yaml.template#L59

Reply:
added